core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -227,7 +227,8 @@ sealed class Vec[T <: Data] private[chisel3] (gen: => T, val length: Int) extend
 
     // Since all children are the same, we can just use the sample_element rather than all children
     // .get is safe because None means mixed directions, we only pass 1 so that's not possible
```

Comment:
we should probably delete this comment since `.get` is apparently not safe

Reply:
Good catch, removed.